.gitignore

```diff
@@ -1,4 +1,5 @@
 # Include only files generated during build, and avoid IDE specific files 
 target/
 site-content
+*.iml
```

Comment:
Oh, just noticed this one. It has to be reverted as well.

ASF Commons projects try to prefer global git ignores instead of multiple IDE ignores (less commits, easier to review changes and commits)